game/main.c

```diff
@@ -3903,7 +3908,7 @@ void reset_gameplay_area()
 	one_vram_buffer(0x8, get_ppu_addr(2, 4 << 3, 9 << 3));
 
 	display_lines_cleared();
-	display_score();
+	multi_vram_buffer_horz("  00000", 7, get_ppu_addr(cur_nt, 0, 6 << 3)); // clear old score
```

Comment:
Why "00000"? You want to have the score default to that instead of empty?

Reply:
Looking into this a bit more. The reason the right most digit was not cleared in the old code (it only wrote 6 digits, not 7) is because the score is always a multiple of 10 (so right most digit is always 0 so it doesn't need to be cleared).

I am not sure if that is still the case in this version of the game, but either way, clearing all 7 is probably a safer bet.

Reply:
> Why "00000"? You want to have the score default to that instead of empty?
> 
> Edit: Oh I think I can guess. That is the default value in the authored version of the screen, and you are just matching that. I think that might mean that after you clear a line, the display will change from "00000" to something like just "40" which doesn't sound ideal.

The new score display routine doesn't clear out the score area, so any 0 tiles will remain in the display until they're overwritten.
E.g. 00000 -> 00040